### PR TITLE
fix: Change useSnaps logic to be based on component size

### LINKED ISSUE #5029: android navigation bar overlaps the "Call" button in the modal call start window
### Description:

When you try to start a call on Android and a modal window appears, the "Call" button is blocked by the Android navigation bar, which makes it impossible to click on it and start a call.

### Environment Information:
- Rocket.Chat.ReactNative Version: 4.37.0
- Device Name: Realme X3 Superzoom
- OS Version: Android 12

### Steps to reproduce:

1. Click on Call button in header room panel
2. You will see that the call button is blocked by the Android navigation bar
3. You will not be able to start call this way

### Expected behavior:

Call button is not overlapped.

### Actual behavior:

It's overlapped.

### Additional context:

Here you can see the screenshot of this behavior:

photo_2023-04-13_16-30-09

Hope for rapid solution. Thanks in advance!

### PR DESCRIPTION
**Pull request by GleidsonDaniel**
## Proposed changes

Change the logic of useSnaps hook to be according to component size and not on percentage

- Tested on Android and iOS devices and tablets

## Issue(s)	

resolves #5029

## Screenshots
<details>
      <summary>Before</summary>
      
Simulator Screen Shot - iPad (10th generation) - 2023-04-13 at 16 52 16
Screenshot 2023-04-13 at 16 46 50
Screenshot 2023-04-13 at 16 48 01
Simulator Screen Shot - iPad (10th generation) - 2023-04-13 at 16 52 49
Screenshot 2023-04-13 at 16 50 49
Screenshot 2023-04-13 at 16 51 06

</details>

<details>
      <summary>After</summary>
      
Screenshot 2023-04-13 at 16 45 14
Screenshot 2023-04-13 at 16 47 16
Simulator Screen Shot - iPad (10th generation) - 2023-04-13 at 16 52 31
Simulator Screen Shot - iPhone 14 - 2023-04-13 at 16 56 13
Screenshot 2023-04-13 at 11 34 52
Simulator Screen Shot - iPhone 14 - 2023-04-13 at 16 56 26
Simulator Screen Shot - iPad (10th generation) - 2023-04-13 at 16 52 41
Screenshot 2023-04-13 at 16 46 02
Screenshot 2023-04-13 at 16 47 39
Screenshot 2023-04-13 at 16 49 58

</details>

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

Jira: [VC-99]

[VC-99]: https://rocketchat.atlassian.net/browse/VC-99?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ankar84** [comment]
Waiting that PR to be merged!

**Comment by ankar84** [comment]
We can't release our android versions with call bug that fixed by that PR

**Review by reinaldonetof — CHANGES_REQUESTED** [review]

File: app/lib/hooks/useVideoConf.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const useVideoConf = (rid: string): { showInitCallActionSheet: () => Prom
 	const isServer5OrNewer = compareServerVersion(serverVersion, 'greaterThanOrEqualTo', '5.0.0');
 
 	const { showActionSheet } = useActionSheet();
-	const snaps = useSnaps([1250]);
+	const snaps = useSnaps(404);
```
[the remark this answers is not in the document]
Comment (GleidsonDaniel) [reply app/lib/hooks/useVideoConf.tsx:39]: The component is 404 px

**Comment by arpsyapathy** [comment]
We are waiting this PR too. Cant build apps.

**Review by reinaldonetof — APPROVED** [review]

**Comment by marceloschmidt** [comment]
@diegolmello what version is this merged into?

**Comment by diegolmello** [comment]
@marceloschmidt 4.39.0. It's not released yet.

**Comment by ankar84** [comment]
Tested - looks good! Thank you, team!!!

**Pull request merged**